tests/api/v1/datadog/api_usage_metering_test.go

```diff
@@ -317,6 +317,10 @@ func TestUsageBillableSummary(t *testing.T) {
 }
 
 func TestSpecifiedDailyCustomReports(t *testing.T) {
+	if tests.GetRecording() == tests.ModeReplaying {
+		t.Skip("No reports are available yet for Org 321813")
+	}
```

Comment:
can we move it inside error handling?

```go
if err != nil {
	if tests.GetRecording() != tests.ModeReplaying && httpresp.StatusCode == 404 {
		t.Skip("No reports are available yet")
	} else {
		t.Errorf("Error getting ...")
	}
}
```

Reply:
These tests are scheduled to run every 3 hours so eventually we should see a successful trace.

Reply:
Done